packages/ripple-binary-codec/test/definitions.test.js

```diff
@@ -0,0 +1,71 @@
+const { encode, decode } = require('../dist')
```

Comment:
This file should be in TypeScript - e.g. `definitions.test.ts`

Reply:
Why should this file be in TypeScript? (The rest of the test files are js for the binary codec)

Reply:
There's a decent amount of config changes required to make ts files work with our setup (We have to update our eslint settings, change the tsconfig, update our run script in npm)

After spending about 30 minutes trying to get it working, I don't think this is worth doing right now. I can make a follow up ticket for it, but for this test in particular it doesn't add much error checking because we're violating the type checks anyway in order to update the `definitions` mid-test.